File: myintent/src/main/java/pl/mareklangiewicz/myintent/MIHelpFragment.java
package pl.mareklangiewicz.myintent;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.webkit.WebSettings;

import pl.mareklangiewicz.myfragments.MyWebFragment;

/**
 * Created by Marek Langiewicz on 13.10.15.
 */
public class MIHelpFragment extends MyWebFragment {

    @SuppressLint("SetJavaScriptEnabled") @Nullable @Override public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {

        View rootView = super.onCreateView(inflater, container, savedInstanceState);

        //noinspection ConstantConditions
        WebSettings webSettings = mWebView.getSettings();
        webSettings.setUserAgentString(webSettings.getUserAgentString() + " " + getString(R.string.mi_user_agent_suffix));
        webSettings.setJavaScriptEnabled(true);

        if(getURL() == null)
            setURL("file:///android_asset/mi.html");


        return rootView;
    }
}

File: myintent/src/main/java/pl/mareklangiewicz/myintent/MIHelpFragment.kt
package pl.mareklangiewicz.myintent

import android.annotation.SuppressLint
import android.os.Bundle
import android.view.View
import pl.mareklangiewicz.myfragments.MyWebFragment

/**
 * Created by Marek Langiewicz on 13.10.15.
 */
class MIHelpFragment : MyWebFragment() {

    @SuppressLint("setJavaScriptEnabled")
    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {

        manager?.name = BuildConfig.NAME_PREFIX + getString(R.string.mi_help)

        if(url == "") url = "file:///android_asset/mi.html"

        settings.userAgentString = settings.userAgentString + " " + getString(R.string.mi_user_agent_suffix)
        settings.javaScriptEnabled = true

        super.onViewCreated(view, savedInstanceState)
    }
}
